src/components/Molecules/SingleMessage/SingleMessage.style.js

```diff
@@ -79,7 +78,18 @@ const Copy = styled.div`
           }
           margin: auto;
           padding: 100px 20px;
-        `}
+          // move empty p tag up with negative margin equal to h1's margin
+          & p:empty {
```

Comment:
I just learnt about `:empty` selector :) .. Can we try to check if it's empty not null in **RichText** component?
```
[BLOCKS.PARAGRAPH]: (node, children) => {
      const fontSize = options.type === 'card' ? 'sm' : 'l';
      **if node !== ""**
      return (
        <Text tag="p" color={changeColor(options.background)} size={fontSize}>
          {children}
        </Text>
      **return null**
      );
    },
```